docs/root/operations/admin.rst

```diff
@@ -190,9 +190,118 @@ The fields are:
 
   .. http:get:: /stats?format=json
 
-  Outputs /stats in JSON format. This can be used for programmatic access of stats.
+  Outputs /stats in JSON format. This can be used for programmatic access of stats. Counters and Gauges
+  will be in the form of a set of (name,value) pairs. Histograms will be under the element "histograms",
+  that contains "supported_quantiles" which lists the quantiles supported and an array of computed_quantiles
+  that has the computed quantile for each histogram. Only histograms with recorded values will be exported.
 
-  .. http:get:: /stats?format=prometheus
+  If a histogram is not updated during an interval, the ouput will have null for all the quantiles.
+  
+  Example histogram output:
+
+.. code-block:: json
+
+  {
+    "histograms": {
+    "supported_quantiles": [
+      0,
+      25,
+      50,
+      75,
+      90,
+      95,
+      99,
+      99.9,
+      100
+    ],
+    "computed_quantiles": [
+      {
+        "name": "cluster.external_auth_cluster.upstream_cx_length_ms",
+        "values": [
+          {
+            "interval": 0,
```

Comment:
for doc legibility purposes, consider this format:
```
        "values": [
           {"interval": 0, "cumulative": 0},
           {"interval": 0, "cumulative": 0},
           {"interval": 1.0435787, "cumulative": 1.0435787},
           ....
```
I assume the whitespace output is not actually what the json serializer actually emits, right?  It would omit all whitespace?